### ISSUE TITLE
Root bone stays anchored with 1.1.1

### ISSUE THREAD

**Issue by Hunanbean** [Rokoko/rokoko-studio-live-blender#53]
version 1.1.1 no longer functions properly when retargeting third party animations, such as mixamo.
1.1.0 had no issues, but with 1.1.1 it insists on leaving the root bone at its starting location.
so, this means my character moves forward, but her hips stay in the starting position.
reverting to the previous version fixes this entirely. all the assets are the exact same ones, the Rokoko plugin was the only variable.
i have tested several times and every time i revert to previous version 1.1.0, everything functions properly again.

thanks and be well

**Comment by Darkblader24** [comment]
Hey, thanks for the report! If you want the root bone to be animated as well you have to match it with the corresponding bone in the bone list.

Also I tried downloading a couple Mixamo animations but none of them had a root bone and they all worked when I tried to retarget them without matching any bones manually. What characters did you use exactly?

**Comment by Hunanbean** [comment]
Yes, i am aware of the procedure, hence the clarification that it works fine with all the same assets, but using 1.1.0 instead of 1.1.1. So unless something fundamental changed between the two, i am pretty sure i did everything identically. I would be glad to upload the mixamo skeleton i was retargeting from. I was retargeting to a MakeHuman Default skeleton. Also, you will find by the comments on youtube, i am not the only one with the issue. 
Thank you for the response, and for being a contributor to the project. Be well

**Comment by Darkblader24** [comment]
Yes something changed indeed, the hips bone won't be matched with the root bone anymore automatically. This is probably the reason why it doesn't get retargeted. So to fix this you just have to manually match the root bone with the hips bone in the bone list.

This was done to prevent cases where the hips bone got matched with the root bone unintentionally. I will look into making both cases work. It would be great if you could send me that character for testing! Is the Mixamo character the one with the root bone or the MakeHuman one?

**Comment by Hunanbean** [comment]
Couple of options. You can experience what i mean by downloading the mixamo animation as a pack, instead of individual where you can set root motion. Female Locomotion pack, tied to Alyssa Character will be what I would send you. To be honest, I thought I had everything mapped correctly, but I may have missed something. It takes a bit for me to save and revert everything for the sake of testing, but i will let you know what i see. The MakeHuman character does automatically add/import a root bone, of sorts. A bit difficult to explain unless you see the Outliner in front of you. So, summary..
Mixamo Package = Female Locomotion Pack (the pack, not the individual download) Attached to Alyssa Character (not really required to be bound to that character)
Blender/MakeHuman = Any character you make in MakeHuman, including generic. Tested with Default skeleton from MakeHuman. Transfered over by either "MakeHuman Plugin For Blender, MPFB" or MHX2 export/import. IIRC the end result is the same with either method.

Thank you

Also, i would need to know which email you would like me to send the MH model to (not a fan of the 3rd party dropbox type solutions)

**Comment by Darkblader24** [comment]
I see, thanks for the explanation! You can send me the file to philipp.koehler24@web.de
